flexget/api/app.py

```diff
@@ -199,7 +224,7 @@ def pagination_parser(self, parser=None, sort_choices=None, default=None, add_so
         'status': {'type': 'string'},
     },
     'required': ['status_code', 'message', 'status'],
-}
+}  # type: MessageDict
```

Comment:
was this comment left on purpose?

Reply:
`MessageDict` is a defined within the `if TYPE_CHECKING` block, and therefore not "available" during execution (as that part would not have been run). Therefore, when using annotations based on TypeDicts, I'm using this syntax instead of `base_message: MessageDict = { ... }`, to avoid errors in python <3.8

Reply:
Couldn't it be `base_message: "MessageDict" = { ... }`? That seems nicer to me than the comment version.

Reply:
@gazpachoking as per my comment above, using that annotation type for this instance would crash, as it's defined within the `TYPE_CHECKING` block (which is necessary because I'm using TypedDict and that only exists in Python 3.8+). 

Minified example:
```python
❯ bat foo.py
───────┬──────────────────────────────────────────────
       │ File: foo.py
───────┼──────────────────────────────────────────────
   1   │ from typing import TYPE_CHECKING
   2   │
   3   │ if TYPE_CHECKING:
   4   │     from typing import TypedDict
   5   │     TD = TypedDict('TD', {'foo': int})
   6   │
   7   │ d1 = {'foo': 3}  # type: TD
   8   │
   9   │ d2: TD = {'foo': 4}
───────┴──────────────────────────────────────────────
❯ mypy foo.py
Success: no issues found in 1 source file
❯ python foo.py
Traceback (most recent call last):
  File "foo.py", line 9, in <module>
    d2: TD = {'foo': 4}
NameError: name 'TD' is not defined
```

Reply:
Notice the quotes in my version. That makes the type annotation a string, not a runtime reference to the type.

Reply:
Sorry, I missed that. Yeah, we could go with that version since you prefer it, updated in branch